src/static/tutorials.json

```diff
@@ -339,5 +339,43 @@
         "description": "Explore how CIDs are constructed in Filecoin, from multihash and multicodec prefixes to CID versions. (No coding required.)"
       }
     ]
+  },
+  "0008": {
+    "id": 8,
+    "formattedId": "0008",
+    "url": "introduction-to-libp2p",
+    "project": "libp2p",
+    "title": "Introduction to libp2p",
+    "description": "In this tutorial you'll learn what libp2p is, what problems it is trying to solve, and how it tries to solve them. You'll also get an overview of it's modularity and how it aims to be the de facto peer-to-peer library.",
+    "newMessage": "",
+    "updateMessage": "",
+    "createdAt": "2020-11-16T18:04:37.607Z",
+    "updatedAt": "2020-11-16T18:04:37.607Z",
+    "resources": [
+      {
+        "title": "libp2p main website",
+        "link": "https://libp2p.io",
+        "type": "docs",
+        "description": "libp2p main website. Includes links to all the native implementations, videos of past talks, specifications and documentation."
+      },
+      {
+        "title": "libp2p documentation website",
+        "link": "https://docs.libp2p.io/",
+        "type": "docs",
+        "description": "Includes code examples and concepts glossary."
+      },
+      {
+        "title": "Introduction to libp2p by David Dias",
+        "link": "https://www.youtube.com/watch?v=CRe_oDtfRLw",
+        "type": "video",
+        "description": "Talk that heavily inspired this tutorial. David goes through why we need libp2p, how it works and what are its objectives for the future."
+      },
+      {
+        "title": "libp2p javascript code examples",
+        "link": "https://github.com/libp2p/js-libp2p/tree/master/examples",
+        "type": "docs",
+        "description": "Examples include a chat app, browser, encrypted communications, peer and content routing and more."
+      }
```

Comment:
I suggest we add here for JS:
- https://github.com/libp2p/js-libp2p/blob/master/doc/GETTING_STARTED.md
- https://github.com/libp2p/js-libp2p/blob/master/doc/CONFIGURATION.md

Reply:
done! 89e0c16a651d77af2672c17757a81764a922b635